features/step_definitions/basic_steps.rb

```diff
@@ -25,3 +21,12 @@
     create(:user)
   end
 end
+
+Given("I open the browser") do
```

Comment:
This name is a bit misleading. I would go with 
```ruby
Then("show me the page") do
```

Reply:
That's already fixed